modules/cnf-topology-aware-lifecycle-operator-operator-and-platform-update.adoc

```diff
@@ -0,0 +1,136 @@
+[id="talo-operator-and-platform-update_{context}"]
+= Performing a platform and an Operator update together
+
+You can perform a platform and an Operator update at the same time.
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Update ZTP to the latest version.
+* Provision one or more managed clusters with ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Create {rh-rhacm} policies in the hub cluster.
+
+.Procedure
+
+. Create the `PolicyGenTemplate` CR for the updates by following the steps described in the "Performing a platform update" and "Performing an Operator update" sections.
+
+. Apply the prep work for the platform and the Operator update.
+
+.. Save the content of the `ClusterGroupUpgrade` CR with the policies for platform update preparation work, catalog source updates, and target clusters to the `cgu-platform-operator-upgrade-prep.yml` file, for example:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-platform-operator-upgrade-prep
+  namespace: default
+spec:
+  managedPolicies:
+  - du-upgrade-platform-upgrade-prep
+  - du-upgrade-operator-catsrc-policy
+  clusterSelector:
+  - group-du-sno
+  remediationStrategy:
+    maxConcurrency: 10
+  enable: true
+----
+
+.. Apply the `cgu-platform-operator-upgrade-prep.yml` file to the hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f cgu-platform-operator-upgrade-prep.yml
+----
+
+.. Monitor the process. Upon completion, ensure that the policy is compliant by running the following command:
++
+[source,terminal]
+----
+$ oc get policies --all-namespaces
+----
+
+. Create the `ClusterGroupUpdate` CR for the platform and the Operator update with the `spec.enable` field set to `false`.
+.. Save the contents of the platform and Operator update `ClusterGroupUpdate` CR with the policies and the target clusters to the `cgu-platform-operator-upgrade.yml` file, as shown in the example:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-du-upgrade
+  namespace: default
+spec:
+  managedPolicies:
+  - du-upgrade-platform-upgrade <1>
+  - du-upgrade-operator-catsrc-policy <2>
+  - common-subscriptions-policy <3>
+  preCaching: true
+  clusterSelector:
+  - group-du-sno
+  remediationStrategy:
+    maxConcurrency: 1
+  enable: false
+----
+<1> This is the platform update policy.
+<2> This is the policy containing the catalog source information for the Operators to be updated. It is needed for the pre-caching feature to determine which Operator images to download to the spoke cluster.
+<3> This is the policy to update the Operators.
+
+.. Apply the `cgu-platform-operator-upgrade.yml` file to the hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f cgu-platform-operator-upgrade.yml
+----
+
+. (Optional) Pre-cache the images for the platform and the Operator update.
```

Comment:
Here and elsewhere, I suggest using our more common usage for formatting "Optional"
```suggestion
. :Optional: Pre-cache the images for the platform and the Operator update.
```